<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="progressBar_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="Input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>[Select Data]</string>
    </property>
    <widget class="QComboBox" name="StockComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stocks</string>
     </property>
    </widget>
    <widget class="QLabel" name="StartDate_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="StartDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="EndDate_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>EndDate</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="EndDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="time">
      <time>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QPushButton" name="ConfirmButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>210</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetButton">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>210</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Q1_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Q1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Q2_radioButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Q2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Q3_radioButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Q3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Q4_radioButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Q4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Q0_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>User Defined Date</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>[Select Results]</string>
    </property>
    <widget class="QCheckBox" name="SelectAll_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Candle_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Candle chart</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="LIME_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LIME</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="SHAP_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SHAP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>120</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>120</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Chart_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predict chart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="expButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>150</y>
       <width>170</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Model Explanation</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>190</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="StockInfo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>15</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>[Stock]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>[Period]</string>
     </property>
    </widget>
    <widget class="QLabel" name="Stockname_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>5</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AAPL (Code)</string>
     </property>
    </widget>
    <widget class="QLabel" name="Period_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>25</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>01/01/2000 - 01/01/2002</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="UserMemo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>[Memo]</string>
    </property>
    <widget class="QTextEdit" name="Memo_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>[Widgets]</string>
    </property>
    <widget class="QPushButton" name="calendarButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="infoButton">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>Input</zorder>
   <zorder>progressBar_label</zorder>
   <zorder>progressBar</zorder>
   <zorder>Output</zorder>
   <zorder>ExitButton</zorder>
   <zorder>StockInfo</zorder>
   <zorder>UserMemo</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
